### PR TITLE
Clearly tell user that E2EE has been enabled for an account

### PR DESCRIPTION
**Pull request by claucambra**
Screenshot 2022-11-15 at 12 26 51

Screenshot 2022-11-15 at 12 26 54

Part of https://github.com/nextcloud/desktop/issues/5126

Signed-off-by: Claudio Cambra <claudio.cambra@nextcloud.com>

### CODE REVIEW

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Nice! :) Just the icons are a bit off:
- The checkmark icon shouldnt be the "All synced" icon
- The info icon looks a bit oldish

Similar like for "General" and "Network", could we use native icons for each of them which work for the platform?

Otherwise as quick fixes:
- E2E active, a lock: https://fonts.google.com/icons?selected=Material%20Icons%3Alock%3A
- E2E possible, info i: https://fonts.google.com/icons?selected=Material%20Icons%3Ainfo%3A

File: src/gui/accountsettings.cpp [hunk 1]
```diff
@@ -246,9 +246,12 @@ void AccountSettings::slotE2eEncryptionMnemonicReady()
     connect(actionDisplayMnemonic, &QAction::triggered, this, [this]() {
         displayMnemonic(_accountState->account()->e2e()->_mnemonic);
     });
-    _ui->encryptionMessage->setText(tr("This account supports End-to-End encryption"));
+
+    _ui->encryptionMessage->setMessageType(KMessageWidget::Positive);
+    _ui->encryptionMessage->setText(tr("End-to-End encryption has been enabled for this account"));
```
[the remark this answers is not in the document]
Comment (claucambra) [reply src/gui/accountsettings.cpp:251]: Yes, I think so, as the server supports several types of encryption systems and "encryption" is not clear which one

**Comment by claucambra** [comment]
> Nice! :) Just the icons are a bit off:
> 
> * The checkmark icon shouldnt be the "All synced" icon
> * The info icon looks a bit oldish
> 
> Similar like for "General" and "Network", could we use native icons for each of them which work for the platform?
> 
> Otherwise as quick fixes:
> 
> * E2E active, a lock: https://fonts.google.com/icons?selected=Material%20Icons%3Alock%3A
> * E2E possible, info i: https://fonts.google.com/icons?selected=Material%20Icons%3Ainfo%3A

Okay, I have changed the icons used in the message box and updated the screenshots above 🙂

Ad for native icons more generally, let's open a separate issue for this

File: src/gui/accountsettings.cpp [hunk 2]
```diff
@@ -246,9 +246,12 @@ void AccountSettings::slotE2eEncryptionMnemonicReady()
...
+    _ui->encryptionMessage->setIcon(Theme::createIconFromSvgResource(QStringLiteral(":/client/theme/colored/state-ok.svg")));
     _ui->encryptionMessage->show();
-    
+
```
[the remark this answers is not in the document]
Comment (claucambra) [reply src/gui/accountsettings.cpp]: Removed

[the remark this answers is not in the document]
Comment (claucambra) [reply src/gui/accountsettings.cpp]: removed

File: src/gui/accountsettings.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (claucambra) [reply src/gui/accountsettings.cpp:251]: Then again, if it is a contextual string (e.g. a messagebox message) that should be clear enough

[the remark this answers is not in the document]
Comment (jancborchardt) [reply src/gui/accountsettings.cpp:251]: Btw, the wording or capitalization should be End-to-end encryption, with lowercase second e. And if in a sentence, the first e should be lowercase as well of course.

[the remark this answers is not in the document]
Comment (claucambra) [reply src/gui/accountsettings.cpp]: Thanks, this is now done :)

**Review by jancborchardt — APPROVED** [review]
Noice! :)

**Comment by tobiasKaminsky** [comment]
Not sure if it is still valid, but in second screenshot:
I assume that account has no E2E yet, so button might better be "setup" instead of "enable e2e" to distinguish this more.

**Comment by claucambra** [comment]
> Not sure if it is still valid, but in second screenshot:
> I assume that account has no E2E yet, so button might better be "setup" instead of "enable e2e" to distinguish this more.

changed :)

**Pull request merged**